mod-source-record-manager-server/src/test/java/org/folio/services/JobExecutionProgressServiceImplTest.java

```diff
@@ -102,14 +104,15 @@ public void shouldNotReturnFailedFutureWhenJobExecutionIdDuplicates(TestContext
     Async async = context.async();
     int expectedTotalRecords = 62;
 
-    Future<JobExecutionProgress> future = jobExecutionService.initializeJobExecutions(initJobExecutionsRqDto, params)
-      .compose(initJobExecutionsRsDto -> jobExecutionProgressService.initializeJobExecutionProgress(initJobExecutionsRsDto.getParentJobExecutionId(), expectedTotalRecords, TENANT_ID)
-        .compose(jobExecutionProgress -> jobExecutionProgressService.initializeJobExecutionProgress(jobExecutionProgress.getJobExecutionId(), expectedTotalRecords, TENANT_ID)));
+    Future<CompositeFuture> future = jobExecutionService.initializeJobExecutions(initJobExecutionsRqDto, params)
+      .compose(initJobExecutionsRsDto -> {
+        Future<JobExecutionProgress> future1 = jobExecutionProgressService.initializeJobExecutionProgress(initJobExecutionsRsDto.getParentJobExecutionId(), expectedTotalRecords, TENANT_ID);
+        Future<JobExecutionProgress> future2 = jobExecutionProgressService.initializeJobExecutionProgress(initJobExecutionsRsDto.getParentJobExecutionId(), expectedTotalRecords, TENANT_ID);
```

Comment:
aren't they identical?

Reply:
yes, they are identical, two concurrent attempts to create progress with the same jobExecution id

Reply:
oops, you're right, missed that it's a test